app/__init__.py

```diff
@@ -62,3 +62,9 @@ def inject_environment():
 @app.before_request
 def load_user():
     g.current_user = getLoginUser()
+
+from app.logic.loginManager import getCurrentTerm
+@app.before_request
+def load_currentTerm():
+
```

Comment:
I'd get rid of this newline, since this function is only one line to begin with. Keep the one-liner next to the definition